### ISSUE TITLE
[BUG] Broken Menu Screen

### ISSUE THREAD

**Issue by kirifto** [bloxstraplabs/bloxstrap#3351]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### What problem did you encounter?

I have had to discontinue the usage of Bloxstrap due to the fact that when I enable it, the menu that you use to leave or reset is completely broken. The only UI that shows up is the Reset, Leave, and Resume buttons. I cannot ALT + F4 to leave the ROBLOX application, which forces me to taskkill in command prompt or go to task manager to end task. I have tried looking through the settings numerous times, and have also tried fixing it myself. Reinstalling doesn't do anything, and completely wiping Bloxstrap and all of its data off of my computer doesn't do anything either.

_TL;DR, Bloxstrap is bugging the Leave/Reset UI and I cannot leave the game even when I try ALT + F4._

### Bloxstrap Log

```text
N/A
```

**Comment by pizzaboxer** [comment]
Are you using the V1 menu? If so, this issue is addressed in the Wiki
